app/assets/javascripts/Components/Result/image_viewer.jsx

```diff
@@ -6,15 +6,30 @@ export class ImageViewer extends React.Component {
   constructor() {
     super();
     this.state = {
-      rotation: 0,
-      zoom: 1,
+      rotation: window.start_image_rotation || 0,
+      zoom: window.start_image_zoom || 1,
       heightChange: 0,
```

Comment:
Looks like `heightChange` and `widthChange` aren't used any more, so you can delete them.